styles/home.css

```diff
@@ -1,62 +1,258 @@
-.nav {
+/*
+------------------------------------
+    FULL PAGE
+------------------------------------
+*/
+
+html, body {
+    height: 100%;
+    width: 100%;
     background-color: #17243A;
-    color: white;
-    height: 5em;
 }
 
-#logo {
-    padding-left: 2em;
-    color: white;
+/*
+------------------------------------
+    WRAPS THE CONTENT
+------------------------------------
+*/
+
+.container {
+    margin: 0;
+    padding: 10vw;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
 }
 
-#name {
-    color: white;
-    text-align:right;
-    padding-right: 2em;
+.content {
+    height: 60%;
+    width: 100%;
+    display: flex;
+    align-items: center;
 }
 
-#image-dir {
-    border: 1px solid black;
+.vcenter {
+    display: flex;
+    align-items: center;
 }
 
-#task-list {
-    border: 1px solid black;
-    height: 10em;
+/*
+------------------------------------
+    LINE SEPARATOR ON BIG SCREENS
+------------------------------------
+*/
+
+.sidebar {
+    height: 60%;
 }
 
-#new-batch, #continue-batch {
-    margin: auto;
-    width: 60%;
-    font-size: 2rem;
-    text-align: center;
-    font-family: "Open Sans", sans-serif;
-    border-radius: 1em;
-    line-height: 3rem;
-    transition: all 0.3s cubic-bezier(.25,.8,.25,1);
+.line {
```

Comment:
If you make this `border-left` and make it a property of the container div, could it expand with the rest of the flexbox?